rviz_default_plugins/include/rviz_default_plugins/displays/image/ros_image_texture.hpp

```diff
@@ -127,6 +127,12 @@ class ROSImageTexture : public ROSImageTextureIface
   ImageData setFormatAndNormalizeDataIfNecessary(ImageData image_data);
   void loadImageToOgreImage(const ImageData & image_data, Ogre::Image & ogre_image) const;
 
+  void imageConvertYUV422ToRGB(
+    uint8_t * dst_img, uint8_t * src_img,
+    int dst_start_row, int dst_end_row,
+    int dst_num_cols, uint32_t stride_in_bytes,
+    std::string src_format);
```

Comment:
As far as I can tell, this is a true function, and as such doesn't need to be a member function of the class.  I'll suggest to remove it from the class and just make it an anonymous function in the C++ file instead.